src/test/java/com/deque/axe/android/wrappers/AxeViewBuilder.java

```diff
@@ -161,10 +169,19 @@ public String text() {
     return text;
   }
 
+  public AxeViewBuilder hintText(final String newValue) {
```

Comment:
Are these changes necessary for this pull request? If so, I'd like them as a separate pull request with their own associated issue before this one is merged in. There is no way I would be able to backtrack and find this work a year later, when the merge for this PR will be "calculated props"

Reply:
Agreed, I'll close this PR and create PRs separately.

Reply:
This for testing purpose.